### DISCUSSION TITLE
Add LT7683+ Support

### DISCUSSION THREAD

**Discussion by Criomod** [Bodmer/TFT_eSPI#1666] [New processor/display support or capability request]
Has anyone considered adding support for the ER-M070-7 from buydisplay.com with the LT7683+ Controller?

https://www.buydisplay.com/low-cost-7-inch-tft-touch-display-module-for-arduino-i2c-spi-interface

Thanks,
Criomod

**Comment by Bodmer** [comment]
That looks like a rather specialist display with built in graphics functions.

Adding new displays that use different command structures tends to be very error prone and time consuming, and as I do not have one of those I will not be able to help. I will flag as help needed.

**Comment by phanthaitrung** [comment]
Hi, just clear that is not a specialist / built in function, but a popular, common SPI display. 

I think LT768x is similar to the popular ILI9xxx driver chip.

I have plan to order one, because it support SPI & 8 inch 800 x 600 & capacitive touch that is rarely not many display suppliers provide. When arrived, I will try to adapt this LT768x to LVGL.
